### PR TITLE
Add `to_pem()` and `from_pem()` for `CmsContentInfo`

### PR DESCRIPTION
**Pull request by application-developer-DA**
Part of the work done for https://github.com/sfackler/rust-openssl/issues/1126

### CODE REVIEW

File: openssl/src/cms.rs [hunk 1]
```diff
@@ -97,13 +97,26 @@ impl CmsContentInfoRef {
...
+        ///
+        /// OpenSSL documentation at [`i2d_CMS_ContentInfo`]
+        ///
+        /// [`i2d_CMS_ContentInfo`]: https://www.openssl.org/docs/man1.0.2/crypto/i2d_CMS_ContentInfo.html
+        to_der,
+        ffi::i2d_CMS_ContentInfo
+    }
+}
+
+#[cfg(ossl101)]
```
Comment (sfackler) [inline openssl/src/cms.rs]: The entire cms module is already `#[cfg(not(libressl))]` so you don't need the extra cfg here.

Comment (application-developer-DA) [reply openssl/src/cms.rs]: Oh, you're right, I did not notice that.

Comment (application-developer-DA) [reply openssl/src/cms.rs]: Updated. Now the AppVeyor build fails, but apparently it has nothing to do with the code...

**Comment by sfackler** [comment]
Thanks!

**Pull request merged**